data.sql

```diff
@@ -38,4 +38,55 @@ VALUES
 (8,'Angemon','2005-06-12',45,true,1),
 (9,'Boarmon','2005-06-07',20.4,true,7),
 (10,'Blossom','1998-10-13',17,true,3),
-(11,'Ditto','2002-05-14',22,true,4);
\ No newline at end of file
+(11,'Ditto','2002-05-14',22,true,4);
+
+-- DAY 3
+
+-- INSER DATA IN owners TABLE
+
+INSERT INTO owners 
+VALUES
+(DEFAULT,'Sam Smith',34),
+(DEFAULT,'Jennifer Orwell',19),
+(DEFAULT,'Bob',45),
+(DEFAULT,'Melody Pond',77),
+(DEFAULT,'Dean Winchester',14),
+(DEFAULT,'Jodie Whittaker',38);
+
+-- INSERT DATA IN Digimon TABLE
+
+INSERT INTO species
+VALUES
+(DEFAULT, 'Pokemon'),
+(DEFAULT, 'Digimon');
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add descriptive data to `species` => `INSERT INTO` and `VALUES` like a below:

 ```sql
 INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');
```